app/lib/router/router.dart

```diff
@@ -170,27 +168,6 @@ List<StatefulShellBranch> mobileShellBranches(Ref ref) {
   ];
 }
 
-List<StatefulShellBranch> desktopShellBranches(Ref ref) {
```

Comment:
I doubt on this changes to make common branches structure as we have different indexing for tabs in mobile and desktop.

Reply:
the tabs shouldn't not be relying the internal branch-index order to just be matching up, it doesn't make any sense. there is an enum now specifying each index in a typed fashion.